PreFreeSurfer/scripts/ACPCAlignment.sh

```diff
@@ -129,9 +102,7 @@ ${FSLDIR}/bin/convert_xfm -omat "$WD"/full2std.mat -concat "$WD"/roi2std.mat "$W
 
 # Get a 6 DOF approximation which does the ACPC alignment (AC, ACPC line, and hemispheric plane)
 verbose_echo " --> Geting a 6 DOF approximation"
-$CARET7DIR/wb_command -convert-affine -from-flirt "$WD"/full2std.mat "$Input".nii.gz "$Reference" -to-world "$WD"/full2std_world.mat
-${HCPPIPEDIR}/global/scripts/aff2rigid_world "$WD"/full2std_world.mat "$WD"/full2std_rigid_world.mat
-$CARET7DIR/wb_command -convert-affine -from-world "$WD"/full2std_rigid_world.mat -to-flirt "$OutputMatrix" "$Input".nii.gz "$Reference"
+${FSLDIR}/bin/aff2rigid "$WD"/full2std.mat "$OutputMatrix"
```

Comment:
This looks like it used a version that didn't include a recent commit.  Please put the "-convert-affine, aff2rigid_world, -convert-affine" sequence back into it.  This probably happened because the working copy was kept outside of the repo, as a "stash, update and reset, stash apply" sequence would have done this properly.

Reply:
This issue is more than cosmetic, but may not affect testing.

Reply:
Done.

Reply:
You need to also remove the line that they replaced (`${FSLDIR}/bin/aff2rigid "$WD"/full2std.mat "$OutputMatrix"`), otherwise it overwrites the good version with the bad one.